.gitignore

```diff
@@ -184,3 +184,6 @@ com_crashlytics_export_strings.xml
 crashlytics.properties
 crashlytics-build.properties
 fabric.properties
+/Brick+extensions.ttl
+/Brick.ttl
+/test.ttl
```

Comment:
Can we remove `test.ttl` from this `.gitignore` file and instead remove the corresponding line from `tests/test_hierarchy_inference.py`? I think we should just stop generating the `test.ttl` file.

None of the files should require the leading `/`; listing, for example, just `Brick.ttl` in the `.gitignore` file is fine.